__tests__/app.test.js

```diff
@@ -82,30 +82,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
-// describe("GET /api/articles", () => {
-//   test("200: Responds an articles array of article objects,", () => {
-//     return request(app)
-//       .get("/api/articles")
-//       .expect(200)
-//       .then(({body})=>{
-//           const {article} = body
-          
-//         expect(Array.isArray(article)).toBe(true);
-//         article.forEach(obj => {
-          
-//           expect(obj).toEqual(
-//             expect.objectContaining({
-//                 article_id: expect.any(Number),
-//                 title: expect.any(String),
-//                 topic: expect.any(String),
-//                 author: expect.any(String),
-//                 created_at: expect.any(String),
-//                 votes: expect.any(Number),
-//                 article_img_url: expect.any(String),
-//                 comment_count: expect.any(Number),
-//             })
-//           )
-//         })
-//       })
-//   });
-// })
\ No newline at end of file
+describe("GET /api/articles", () => {
+  test("200: Responds an articles array of article objects,", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.length).toBe(13);
+        expect(Array.isArray(article)).toBe(true);
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: